pkg/docker/docker.go

```diff
@@ -103,27 +114,66 @@ func (step Step) Exec() (*io.ReadCloser, error) {
 		log.Fatal(err)
 	}
 
+	if len(resp.Warnings) > 0 {
+		for warning := range resp.Warnings {
+			log.Warn(warning)
+		}
+	}
+
 	if err = cli.ContainerStart(ctx, resp.ID, types.ContainerStartOptions{}); err != nil {
 		log.Fatal(err)
 	}
 
-	statusCh, errCh := cli.ContainerWait(ctx, resp.ID, container.WaitConditionNotRunning)
-	select {
-	case err = <-errCh:
+	defer func() {
+		dur, err := time.ParseDuration("-1ns") // Negative duration means no force termination
```

Comment:
Can you explain why we have to do this? It's bit unclear to me

Reply:
Reference: https://godoc.org/github.com/docker/docker/client#Client.ContainerStop
>If the timeout is nil, the container's StopTimeout value is used, if set, otherwise the engine default. A negative timeout value can be specified, meaning no timeout, i.e. no forceful termination is performed.